src/apis/platformvm/api.ts

```diff
@@ -528,6 +529,34 @@ export class PlatformVMAPI extends JRPCAPI {
     )
     return response.data.result
   }
+  /**
+   * List amounts that can be claimed: validator rewards, expired deposit rewards, active deposit rewards claimable at current time.
+   *
+   * @param addresses An array of addresses as cb58 strings or addresses as {@link https://github.com/feross/buffer|Buffer}s
+   * @param depositTxIDs An array of deposit transactions ids
+   * @param locktime Optional. The locktime field created in the resulting outputs
+   * @param threshold Optional. The number of signatures required to spend the funds in the resultant UTXO
+   *
+   * @returns Promise for an object containing the amounts that can be claimed.
+   */
+  getClaimables = async (
+    addresses: string[],
+    depositTxIDs: string[],
+    locktime: BN = ZeroBN,
+    threshold: number = 1
+  ): Promise<object> => {
+    const params: GetClaimablesParams = {
+      locktime,
+      threshold,
+      addresses,
+      depositTxIDs
+    }
+    const response: RequestResponseData = await this.callMethod(
+      "platform.getClaimables",
+      params
+    )
+    return response.data.result
```

Comment:
What response is provided? We usually have structs / interface defined in interfaces.ts if the return result is more than a simple string or number.